src/EnergyPlus/SystemAvailabilityManager.hh

```diff
@@ -456,6 +459,14 @@ namespace SystemAvailabilityManager {
 		// manager is connected to air loop
 		bool SimHybridVentSysAvailMgr; // Set to false when a zone has two hybrid ventilation
 		// managers, one with air loop and one without
+		Real64 OperativeTemp; // Zone air operative temperature
+		Real64 CO2; // Zone air CO2 temperature
```

Comment:
@mjwitte Wow! I did not know this. Thanks for pointing out. I will use it as an output variable. Thanks.